README.md

```diff
@@ -1,12 +1,12 @@
 # [Kata Seeds](https://github.com/kata-seeds): Language (+ optionally a test framework)
-[![Build Status](https://travis-ci.org/kata-seeds/language-seed.svg?branch=master)](https://travis-ci.org/kata-seeds/language-seed)
+[![Build Status](https://travis-ci.org/kata-seeds/kata-seed-template.svg?branch=master)](https://travis-ci.org/kata-seeds/kata-seed-template)
```

Comment:
Yeah, I did that below. In this case (for the travis info), it looks wrong to me.